Common/include/CConfig.hpp

```diff
@@ -521,6 +521,7 @@ class CConfig {
   Kind_Gradient_Method_Recon,      /*!< \brief Numerical method for computation of spatial gradients used for upwind reconstruction. */
   Kind_Deform_Linear_Solver,             /*!< Numerical method to deform the grid */
   Kind_Deform_Linear_Solver_Prec,        /*!< \brief Preconditioner of the linear solver. */
+  Kind_Graph_Part_Algo,    /*!< \brief Algorithm for parallel partitioning of the matrix graph. */
```

Comment:
Align with other comments
```suggestion
  Kind_Graph_Part_Algo,              /*!< \brief Algorithm for parallel partitioning of the matrix graph. */
```